### ISSUE TITLE
Nothing seems to be working on Simcraft 720-02. I cannot click any of the tabs at all, have tried reinstalling it and still the same problems

### ISSUE THREAD

**Issue by emeebia** [simulationcraft/simc#3877]
Please include a brief overview of the issue in the title above, as well as completing the following:

#### simc_gui.simc file or contents
This file is saved in the same folder as the program after each simulation. Make sure to run the simulation that is giving you issues, then include the file here by either uploading to github, copy/pasting the contents, or using http://pastebin.com/

#### Reproduction Steps:

Please include any other steps necessary to recreate the problem, or include logs/data to show what is occurring in game if there is an issue with how something is modeled.

**Comment by lgkern** [comment]
It is probably because you're using 720-02, try using 725-01

**Comment by emeebia** [comment]
Nope have downloaded newest version and still having the same issues

**Comment by slicnic81** [comment]
Im having the same problem, have you been able to fix this emeebia?

**Comment by PauloFranco** [comment]
Having the same problem as you people, on 725-02

**Comment by emeebia** [comment]
I got a email off simcraft a while back, explaining it might be the version I was using and to install the newer version. I have tried that and still the problem persists. Anyone else heard from simcraft? and if so any ideas on what the problem might be?




----Original Message----

**Comment by PauloFranco** [comment]
No idea, as of today I still got this bug

**Comment by vinevicious** [comment]
i'm having the same problem, it started when i did a fresh windows install.

**Comment by emeebia** [comment]
Have you found a solution? Or posted on the forum? Me I am still waiting on a reply




----Original Message----

**Comment by vinevicious** [comment]
i just tryied running with the integrade graphics instead the nvidia gpu (i'm using a laptop) and it is working!

**Comment by emeebia** [comment]
Just tried it to and it's also working off  the integrated graphics card




----Original Message----

**Comment by scamille** [comment]
This problem is related to the GUI framework we use and problems with specific computer/graphic cards setups.

Since we have since done a major version upgrade of the GUI library, please try again with a current 8.x release and update your graphic card drivers as well.

If the problem still persists create a new ticket/reopen and include your dxdiag file - Instructions here - https://us.battle.net/support/en/article/Obtaining-System-Files
